html5/js/Client.js

```diff
@@ -1577,11 +1567,7 @@ class XpraClient {
       "encoding.transparency": true,
       "encoding.decoder-speed": { video: 0 },
       "encodings.packet": true,
-      //"encoding.min-speed"		: 80,
```

Comment:
These examples are nice to have.

Reply:
Okay, how does this look?
```
      //skipping some keys
      //ie: "encoding.min-quality"	: 50,
      //ie: "encoding.min-speed"		: 80,
      //ie: "encoding.non-scroll"		: ["rgb32", "png", "jpeg"],
```

Reply:
Is this allowed by the linter?

Reply:
This PR actually isn't linter-driven, just a bunch of manual judgements and polishing. I worry in general about lines of code that look like they were accidentally/temporarily commented-out, but the linter accepts anything here (as-is, prefixed, deleted, etc.).

Reply:
I just added the prefixes to match other "example" comments kept for documentation.